test/unit/ResultsRow.test.js

```diff
@@ -45,29 +50,29 @@ describe('ResultsRow', () => {
 
     // Publication date check
     it('should contain a publication date', () => {
-      expect(component.find('.nypl-results-date').text()).to.equal('513');
+      expect(component.find('.nypl-results-date').first().text()).to.equal('1922.');
     });
 
     // Publication place check
     it('should contain a publication place', () => {
-      expect(component.find('.nypl-results-place').text()).to.equal('England');
+      expect(component.find('.nypl-results-place').first().text()).to.equal('Chicago :');
     });
 
     // Publisher check
     it('should contain markup only when not provided', () => {
-      expect(component.find('.nypl-results-publisher').text()).to.equal('');
+      expect(component.find('.nypl-results-publisher').first().text()).to.equal('The Drake Hotel Company');
     });
   });
 
   describe('EBookList', () => {
     before(() => {
-      const ebooks = results[0]['_source'].instances[0].items;
+      const ebooks = instances[0].items;
       component = shallow(<EBookList ebooks={ebooks} />);
     });
-    it('should have a list with a single line item with a single anchor tag', () => {
+    it('should have a list with lines with a single anchor tag', () => {
```

Comment:
Change the text here to reflect the test for two anchor tags from the result set.

Reply:
ready